src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -1091,8 +1091,8 @@ export type RepositoryProjectPathConfig = {
   projectSlug: string;
   repoId: string;
   repoName: string;
-  integrationId: string;
-  provider: BaseIntegrationProvider;
+  integrationId: number | null;
```

Comment:
@NisanthanNanthakumar we should keep primary key IDs as strings (we do that for other DB entities as well).